lib/dockerfile.ts

```diff
@@ -84,14 +91,64 @@ export class Dockerfile {
 		return stagedGroups;
 	}
 
+	public generateLiveDockerfile(): string {
+		// First, if there's no live cmd, we can just return the
+		// original dockerfile
+		if (!this.liveCmd) {
+			return this.dockerfileContent;
+		}
+
+		let foundLiveCmd = false;
+		let liveDockerfile = '';
+
+		for (const entry of this.parsedDockerfile) {
+			if (entry.name === 'FROM') {
+				// If we haven't encountered the liveCmd yet, we can
+				// forward this as normal. If we have, we generating
+				// the dockerfile, as stages following a liveCmd
+				// should be ignored
+				if (!foundLiveCmd) {
+					liveDockerfile += `${entry.raw}\n`;
+				} else {
+					break;
+				}
+			} else if (entry.name === 'RUN') {
+				// If we've reached a liveCmd, we no longer care
+				// about any run commands, and can skip them,
+				// otherwise we forward them as usual
+				if (!foundLiveCmd) {
+					liveDockerfile += `${entry.raw}\n`;
+				}
+			} else if (entry.name === 'LIVECMD') {
+				foundLiveCmd = true;
+				liveDockerfile += `CMD ${entry.args}\n`;
```

Comment:
`entry.args` could be an array or a map: https://github.com/joyent/node-docker-file-parser/blob/master/index.d.ts#L5

Reply:
Not for this instance, due to us doing the parsing of `livecmd` directives.

Reply:
I'll add a comment though